lib/hash-tree.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+
+function getFileInfos(fullPath) {
+  var stat = fs.statSync(fullPath);
```

Comment:
if we can't stat something because, we should likely just return a `null` fileInfo. If in the future we can stat it, the caches will differ.